### ISSUE TITLE
How to run on local NWB file

### ISSUE THREAD

**Issue by bendichter** [flatironinstitute/neurosift#80]
It would be great if the README had a section on how to run neurosift on a local NWB file.

We have several conversions where we are not yet ready to post the data on DANDI, but we would still like to test neurosift and see how well it can render the data within these files. What is the preferred workflow for running neurosift on a local NWB file?

**Comment by magland** [comment]
The easy answer (easy if it actually worked) would be to host a local file server, serving a directory containing your nwb files, using something like `python -m http.server 8093`. I just tried it... unfortunately it's not that easy. First problem I ran into was that you can't request resources served over http from a domain served over https. There's no good workaround for that, I'm afraid, but I pressed forward and tried hosting neurosift from localhost just to see if that would work. I ran into some other CORS issues, which I resolved. Then other things didn't work as expected.

Long story short, I don't think hosting a local file server is the path forward.

The next two possiblities that come to mind are (a) using the [filesystem access API](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API) of the browser, and (b) creating an electron app.

The former has some limitations, and I am hesitant to go in that direction.

So that leads to an electron app. I know @garrettmflynn has some infrastructure for this. I'll do some poking around to see if there's a simple way forward here...

**Comment by magland** [comment]
There's actually an alternative way to do this without electron or the filesystem API. I'll follow up...

**Comment by magland** [comment]
I got this to work, and it loads the local nwb file really fast!

I'll try to share some instructions by Monday.

**Comment by magland** [comment]
@bendichter try this!

https://github.com/flatironinstitute/neurosift/tree/main/experimental-local-file-access

**Comment by bendichter** [comment]
I got it working! Now I think we just need a simple python script with a command line entry point so we can run something like

```bash
pip install neurosift
```
to install and then

```
neurosift nwb_file.nwb
```
or 
```
neurosift nwb_directory
```
or
```
neurosift -p 6666 nwbfile.nwb
```

**Comment by magland** [comment]
See https://github.com/flatironinstitute/neurosift#quick-instructions-for-viewing-a-local-nwb-file

**Comment by bendichter** [comment]
Works for me!
